moving-keys-between-servers.md

```diff
@@ -0,0 +1,91 @@
+# How to move a gpg key between servers
+
+Sometimes, you need to move a gpg key from one server to another. 
+
+But why?
+
+Let's say, you prepare a site to deploy to staging. You create a gpg key on your local machine, and you export it to the staging server. For the sake of good house keeping, you then remove the key from your local machine. Days, weeks, months elapse - and, now, you (or someone else) needs to deploy the app to production *on a different server*. You need the gpg key to do so, and the key only lives in one place: the staging server. 
+
+Another example. Let's say, you need to migrate an application from its current server to a new server: for this, the gpg key of the application must be included on the new server's keyring. Again, the gpg key only resides on its current server. 
+
+If these situations sound familiar, then learn how to seamlessly and securely move a gpg key to from one destination to another. 
+
+#### 1. On your local machine, add your private SSH key (i.e., id_rsa) to the ssh-agent.
+
+```
+# local machine
+ssh-add ~/.ssh/id_rsa
+```
+
+#### 2. Shell into the "old" server, and in doing so, forward the authentication credentials to this server.
```

Comment:
at the risk of being super annoying: what does it mean to forward the authentication credentials?

Reply:
GOOD QUESTION. It helped me uncover an error in Step 7!